sunrise.py

```diff
@@ -0,0 +1,379 @@
+                      np.pi / 180) * np.cos(ha * np.pi / 180)
+    # alpha = np.arcsin((sin_term+cos_term)*np.pi/180)
+    alpha = np.arcsin((sin_term + cos_term)) * 180 / np.pi
+
+    return alpha
+
+
+def generate_elevation_series(
+        hour_range,
+        lat=lat,
+        lon=lon,
+        utc=utc,
+        timestamps=False):
+    """
+    Creates a time series of elevation angles for a given
+    set of hours or Pandas timestamps.
+
+    Parameters:
+    -----------
+    hour_range : numpy array or pandas series of Timestamps
+        The time period of desired solar elevation angles.
+        Default is a list of hours, a pandas series of
+        timestamps can also be accepted if the timesteps
+        argument is set to True.
+    lat : float
+        The latitude of the location of interest
+    lon : float
+        The longitude of the location of interest
+    utc : integer
+        The time shift from the Coordinated Universal Time.
+        Western longitudes have UTC < 0. Eastern longitudes
+        have UTC > 0.
+    timestamps : boolean
+        Specifies the type of the hour_range.
+        Default is False (hour_range is array of float or int).
+        True if hour_range is a pandas series of Timestamps.
+
+    Returns:
+    --------
+    elevation_angles : list
+        The list of elevation angles corresponding to each
+        hour or timestamp. Units in degrees.
+    """
+    elevation_angles = []
+
+    for hour in hour_range:
+        if timestamps:
+            hour = timestamp_to_hour(hour)
+        time = local_time(hour)
+        et = equation_of_time(hour)
+        dec = declination(hour)
+        LSTM = local_meridian(utc)
+        tc = time_correction(LSTM, et)
+        lst = local_solar_time(time, tc)
+        ha = hour_angle(lst)
+        elangle = solar_elevation(ha, dec)
+        elevation_angles.append(elangle)
+
+    return elevation_angles
+
+
+if __name__ == "__main__":
+    time = 0  # hours since midnight
+    N = 1
+
+    hour = hour_number(N, time)
+
+    et = equation_of_time(hour)
+
+    lstm = local_meridian()
+
+    tc = time_correction(lstm, et)
+
+    lst = local_solar_time(time, tc)
+
+    ha = hour_angle(lst)
+
+    dec = declination(hour)
+
+    elangle = solar_elevation(ha, dec)
+
+    print(f"The time is: {time}")
+    print(f"The hour number is: {hour}")
+    print(f"The equation of time is: {et}")
+    print(f"The local standard time meridian is: {lstm}")
+    print(f"The time correction factor is: {tc}")
+    print(f"The local solar time is: {lst}")
+    print(f"The hour angle is: {ha}")
+    print(f"The elevation angle is: {elangle}")
+
+    rise = 12 - (180 / np.pi) * (1 / 15) * np.arccos(-np.sin(
+    lat * np.pi / 180) * np.sin(
+        dec * np.pi / 180) / (np.cos(lat * np.pi / 180) * np.cos(
+        dec * np.pi / 180))) - tc / 60
+    print(rise)
+
+    dates = pd.date_range(start='1/1/2015', end='7/1/2019', freq='h')[:-2]
+
+    # t = np.arange(0, 8760, 1)
```

Comment:
Are these comments supposed to be here?

Reply:
I think for right now, I'm going to leave it in the file, because it is a good starting point for a code example.
I'll make an issue to create examples that will deprecate this.